common.pri

```diff
@@ -87,7 +103,20 @@ SOURCES = \
 	$${PWD}/src/views/jobs_view.cc \
 	$${PWD}/src/views/objects/delete_objects_dialog.cc \
 	$${PWD}/src/views/session_dialog.cc \
-	$${PWD}/src/views/session_view.cc
+	$${PWD}/src/views/session_view.cc \
+	src/quazip-0.7.1/quazip/JlCompress.cpp \
+	src/quazip-0.7.1/quazip/qioapi.cpp \
+	src/quazip-0.7.1/quazip/quaadler32.cpp \
+	src/quazip-0.7.1/quazip/quacrc32.cpp \
+	src/quazip-0.7.1/quazip/quagzipfile.cpp \
+	src/quazip-0.7.1/quazip/quaziodevice.cpp \
+	src/quazip-0.7.1/quazip/quazipdir.cpp \
+	src/quazip-0.7.1/quazip/quazipfile.cpp \
+	src/quazip-0.7.1/quazip/quazipfileinfo.cpp \
+	src/quazip-0.7.1/quazip/quazipnewinfo.cpp \
+	src/quazip-0.7.1/quazip/quazip.cpp \
+	src/quazip-0.7.1/quazip/unzip.c \
+	src/quazip-0.7.1/quazip/zip.c
```

Comment:
Prefix these lines with $${PWD} like all the other ones.  Same with the header files.